src/app/uploads/image-upload.js

```diff
@@ -77,7 +77,9 @@ export const ImageUpload = () => {
               {fileNames.map((fileName) => (
                 <TableRow key={fileName}>
                   <TableCell style={{ width: "40%", padding: "20px" }}>
-                    {fileName.length > 20 ? fileName.slice(0, 20) + "..." : fileName}
+                    {fileName.length > 20
```

Comment:
No explicit changes besides justification  formatting changes

Reply:
This is the auto-formatted by VS code.